project.clj

```diff
@@ -104,9 +109,7 @@
                               :port 3449
                               :css-dirs ["resources/site/css"]}
                    :env {:is-dev false}
-                   :plugins [[com.cemerick/austin "0.1.4"]
-                             [lein-figwheel "0.1.5-SNAPSHOT"]]}
-             :uberjar {:main kixi.hecuba.controller.main :aot [kixi.hecuba.controller.main]}}
+                   :plugins [[lein-figwheel "0.1.5-SNAPSHOT"]]}}
```

Comment:
current version of lein-figwheel is 0.2.1-SNAPSHOT. Not sure if you want to upgrade this yet though